cli/src/emscripten-utils.js

```diff
@@ -0,0 +1,13 @@
+export function pathify(path) {
+  if (path.startsWith("file://")) {
+    path = path.slice("file://".length);
```

Comment:
There are various cross-platform edge-cases with `file:` URIs being different. Better to use `require('url').fileURLToPath` here.

But, why do we ever get URIs in CLI in the first place?

Reply:
> But, why do we ever get URIs in CLI in the first place?

That’s rollup’s doing :D